src/renderers/webgl/WebGLAnimation.js

```diff
@@ -1,52 +1,48 @@
-function WebGLAnimation() {
+let context = null;
+let isAnimating = false;
+let animationLoop = null;
+let requestId = null;
 
-	let context = null;
-	let isAnimating = false;
-	let animationLoop = null;
-	let requestId = null;
+function onAnimationFrame( time, frame ) {
 
-	function onAnimationFrame( time, frame ) {
+	animationLoop( time, frame );
 
-		animationLoop( time, frame );
+	requestId = context.requestAnimationFrame( onAnimationFrame );
 
-		requestId = context.requestAnimationFrame( onAnimationFrame );
-
-	}
-
-	return {
+}
 
-		start: function () {
+class WebGLAnimation {
```

Comment:
I originally migrated these function scope variables to instance properties, but e2e testing caught an issue where rAF needs explicit binding its callback to `this`.

Moving internal vars to the module scope eliminates binding issues, but I am not sure they are safe there... Let me know if anybody has advice.

For the animation loop, I wanted to avoid fixes like
```js
this._context.requestAnimationFrame( this._onAnimationFrame.bind( this ) );
```
And
```js
this._context.requestAnimationFrame( ( time, frame ) => this._onAnimationFrame );
```

If using the module scope is unsafe, I can go back to instance props and try caching `this._onAnimationFrame.bind( this )` as another instance prop. 

Thanks in advance...